pages/contact/_hero.html

```diff
@@ -1,7 +1,8 @@
 <div class="container">
   <div class="row">
     <div class="col-xs-12 text-center">
-      <h1>Book a Gruntwork Demo</h1>
+      <h1 style="font-size: 2.5em; font-weight: 700; color: #ffffff;">Book a Gruntwork Demo</h1>
+      <h2 style="font-size: 1.5em; font-weight: 400; color: #6c757d;">Speak with a real human about your DevOps Foundations!</h2>
```

Comment:
NIT: on our other pages, we use a plain `<h1>` (with no inline styles) for the heading and `<p class="lead">...</p>` for the sub-heading. Give that a shot!